src/app/components/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import styled from 'styled-components';
+import SearchIcon from '../Icons/SearchIcon';
+
+type SearchBarProps = {
+  searchQuery: string;
+  setSearchQuery: (searchQuery: string) => void;
+  onSubmit: () => void;
+  placeholderText: string;
+  screenreaderLabelText: string;
+};
+
+export default function SearchBar({
+  searchQuery,
+  setSearchQuery,
+  onSubmit,
+  placeholderText,
+  screenreaderLabelText,
+}: SearchBarProps): JSX.Element {
+  return (
+    <FormContainer
+      onSubmit={(event) => {
+        event.preventDefault();
+        onSubmit();
+        console.log(event);
+      }}
+    >
+      <StyledButton type="submit">
+        <SearchIcon height="40" width="40"></SearchIcon>
+      </StyledButton>
+      <ScreenReaderOnlyLabel htmlFor="styledInput">
+        {screenreaderLabelText}
+      </ScreenReaderOnlyLabel>
+      <StyledInput
```

Comment:
I don't think prefix `Styled` gives you an additional value. I would just call it  `Input`.

```suggestion
      <Input
```